src/object.c

```diff
@@ -775,7 +782,7 @@ size_t objectComputeSize(robj *o, size_t sample_size) {
             while((de = dictNext(di)) != NULL && samples < sample_size) {
                 ele = dictGetKey(de);
                 ele2 = dictGetVal(de);
-                elesize += sdsAllocSize(ele) + sdsAllocSize(ele2);
+                elesize += sizeOfStringObject(ele) + sizeOfStringObject(ele2);
```

Comment:
Same as above, ele and ele2 are sds strings, not robj*. So, the existing code looks correct to me.